src/Charts/chart.ts

```diff
@@ -176,7 +182,12 @@ export abstract class Chart {
     return this
   }
 
-  setOptions(options: ChartOptions) {
+  setOptions(options: ChartOptions) {    
+    for (const key of chartKeys) {
```

Comment:
Why is this necessary now?

Reply:
Because you can now pass `setOtions({ y :  { extentFilter: () => false })`. Which must invalidate the stored extent.